content/1_General/Adding_Solution.mdx

```diff
@@ -99,7 +99,7 @@ See [Working With MDX](/general/working-mdx) for additional information.
 
     - Use _tabs_ to indent
     <Info title="Pro Tip">
-	If your code is indented with spaces, use the [Guide Editor](/editor), copy-paste your code in, right-click, click "Command Palette" from the menu that pops up and then search for "Convert Indentation to Tabs". Once you see it, click on it and then copy that new code with the correct indentation into your Github pull request.
+	If your code is indented with spaces, use the [Guide Editor](/editor). Copy-paste your code in, highlight everything then right-click, click "Command Palette" from the menu that pops up and then search for "Convert Indentation to Tabs". Once you see it, click on it and make sure that all spaces were converted to tabs since certain parts of code can still have bugged spaces. If everything looks good, copy that new code with the correct indentation into your Github pull request.
```

Comment:
Why was this changed? The original is better imo

Reply:
it works for me without highlighting anything